.circleci/config.yml

```diff
@@ -180,12 +179,12 @@ jobs:
       - run: make -C build -j4
       - run: make -C build -j4 tests
       - run: make -C build -j2 check
-      - run: make -C build install
-      - run: make -C build package
       - save_cache:
-          key: ccache
+          key: ccache-release-linux-x86_64-{{ .Branch }}-{{ epoch }}
```

Comment:
Is `epoch` current time? Isn't this always unique? When would this actually be a cache hit?

Reply:
Yes - this is the only way to make sure we write out a new cache dump with each build.  Once a cache key is created, it's immutable for 30 days, so each new build needs to write to a new cache.

The real trick here is the `read_cache` rules, which will find the *most recent* cache key that matches the desired pattern.  That's how a cache from the previous build will get used in the next one.

After reading the CircleCI docs a few times, it seems that this is how its intended to work, but it's not _quite_ spelled out clearly, just strongly implied.